common/timing.cc

```diff
@@ -32,36 +32,42 @@ typedef std::map<int, unsigned> DelayFrequency;
 struct Timing
```

Comment:
You should add your name to the copyright header in timing.cc, given you've now written most of it!